test/jdk/java/nio/file/Files/StreamTest.java

```diff
@@ -27,6 +27,7 @@
  * @build PassThroughFileSystem FaultyFileSystem
  * @run testng StreamTest
  * @summary Unit test for java.nio.file.Files methods that return a Stream
+ * @requires (os.family == "linux")
```

Comment:
Hello Brian, the changes to the rest of the tests in this PR look good to me. I think the change to this test is an oversight. This test class has several test methods and it's only the `testProcFile()` which requires to be run on Linux. Adding this `@requires` would mean that we will now skip running the rest of these test methods on other operating systems.

Reply:
> I think the change to this test is an oversight.

@jaikiran You are correct. Thanks for catching this mistake. I will revert that change to `StreamTest`.

Reply:
`StreamTest` change reverted in 87eafb1.